data/risks/networks/42161/stargate.json

```diff
@@ -2,7 +2,7 @@
 	"$schema": "risks",
 	"label": "Stargate",
 	"codeReviewScore": 3,
-	"testingScore": 4,
+	"testingScore": 3,
```

Comment:
Why are we changing the testing scores in this PR if they were changed in this [PR](https://github.com/yearn/ydaemon/pull/177/files)?

Reply:
Because that was into main and dev is the master branch everyone should be making their prs to dev now